common/log.hpp

```diff
@@ -105,7 +105,7 @@ class Logger {
 
 	string name_ {};
 
-	LogLevel level_;
+	LogLevel level_ {LogLevel::Info};
```

Comment:
You are reverting [this fix](https://github.com/mendersoftware/mender/commit/e2721c676686ce1a9d401b0733301c1d28b1e520).

Reply:
Seems ur are right - TIL!